server/repositories/cmsApi.js

```diff
@@ -0,0 +1,37 @@
+const { getEstablishmentUiId } = require('../utils');
+
+class CmsApi {
+  constructor(jsonApiClient) {
+    this.jsonApiClient = jsonApiClient;
+  }
+
+  async primaryMenu(prisonId = 0) {
+    const response = await this.jsonApiClient.get(
+      `/jsonapi/node/landing_page?fields[node--landing_page]=title,field_moj_description,drupal_internal__nid,field_moj_prisons`,
```

Comment:
will look at replacing this using drupal query string building  library in a follow up